Логинов Андрей

AndrewLoginov opened the pull request:
Решение с доп заданием
[Посмотреть решение](https://hruview.herokuapp.com/AndrewLoginov/markup-task-3/master/index.html)

AndrewLoginov committed:
Логинов Андрей

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

mokhov commented:
Вижу две проблемки: нет переноса строк (поэтому текст выглядит рвано) и полоса справа
<img width="418" alt="document 2018-10-29 10-35-39" src="https://user-images.githubusercontent.com/357689/47631129-6d32e800-db66-11e8-8b83-7b2791d490e6.png">

mokhov commented on index.css:
```diff
@@ -0,0 +1,465 @@
+
+.name_header
+{
+    display: grid;
```
Задание можно решить без гридов

AndrewLoginov replied:
Полосу справа убрал. По поводу мягких переносов, не нашел ни одного сайта, который адекватно расставляет мягкие переносы, либо они вообще не работают. Убрал break-all.

mokhov commented on index.css:
```diff
@@ -0,0 +1,465 @@
+.header_h1
+{
+    font-size: 60px;
+    font-family: 'Fascinate Inline', cursive;
```
Можно использовать просто `font`

mokhov commented on index.html:
```diff
@@ -1,9 +1,225 @@
 </head>
+
 <body>
+    <label class="input1" for="name6">Основной размер</label>
```
Зачем здесь классы?

mokhov commented on index.html:
```diff
@@ -1,9 +1,225 @@
+            <p>3.30</p>
+        </div>
+        <div class="text_header">
+            <p>Amazing adventure</p>
```
Это, скорее всего, больше похоже на меню чем на набор текстов

AndrewLoginov replied:
Нужно же газету сверстать было. Я и не подумал о меню. Окей , исправлю

mokhov commented on index.html:
```diff
@@ -1,9 +1,225 @@
+    <div class="main_newspaper">
+        <div class="first_columns">
+            <article class="first_news">
+                <h2 class="news_h2">Lucky horseshoe
```
Не очень понятен смысл в имени класса держать тег, это и не семантично и странно. `news_header` будет гораздо читаемее

AndrewLoginov replied:
Верно, исправлю

mokhov commented on index.html:
```diff
@@ -1,9 +1,225 @@
+                    <cite class="news_p_second">Lewis</cite>
+                </blockquote>
+                <blockquote>
+                    <p class="news_p">In fact, neither in America nor in the Wild
```
Зачем нужен `p` внутри `blockquote`?

AndrewLoginov replied:
На MDN и htmlbook в blockquote везде p ставят. + атрибут  cite автоматически на новую строку переносится. Как лучше все же?

mokhov commented on index.html:
```diff
@@ -1,9 +1,225 @@
+            <h3 class="fourth_news_h3 news_h3">Children`s<br>clothing or ladies
+                oxfords<br>to go at</h3>
+            <h2 class="fourth_news_h2 news_h2">Cost.<br>Shaws.</h2>
+            <img src="img/6.jpg" alt="man">
```
Нет размеров и хорошо бы `alt` почитаемее

AndrewLoginov replied:
Размер width: 100%; height: 100%;  ,  поместил в родительский блок как бэкграунд, чтобы при изменении цветовой темы, была возможность менять цвет картинки.

mokhov reviewed (changes requested):
Есть куда стремиться 🍅

mokhov commented:
И ещё не вижу вот этого условия
Изредка текст пишут:
В
О
Т

Т
А
К

AndrewLoginov committed:
Логинов Андрей

Решение (2)
1.Убрал белую полосу справа, убрал разрыв слов
2.Обьеденил стили текста в font
3.Убрал лишние классы
4.Сделал в шапке из набора текста меню
5.Подправил название классов
6.Убрал p из blockquote
7. Поменял alt у img на более понятные
8. Добавил вертикальный текст.

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
